projects/ngx-d3timeline/src/lib/ngx-d3timeline.component.ts

```diff
@@ -53,10 +56,18 @@ export class NgxD3timelineComponent implements AfterViewInit {
     this.timeline.setTimeOrientation(value);
   }
 
+  @Output() dropped = new EventEmitter<Activity>();
+
   @ViewChild('svgEl') svgEl: ElementRef<SVGElement>;
 
   constructor(public timeline: NgxD3TimelineService) {}
```

Comment:
are you deliberately not naming this as `timelineService` ?

Reply:
Yeah, this component is the entry point for the library. The timeline. 

The service is a facade for the timeline. Didn't see the benefit of calling it ngxD3TimelineService, I think timeline is descriptive enough